datastore/sqlite_datastore.nim

```diff
@@ -33,53 +39,149 @@ type
     containsStmt: ContainsStmt
     deleteStmt: DeleteStmt
     env: SQLite
+    getDataCol: BoundDataCol
     getStmt: GetStmt
     putStmt: PutStmt
     readOnly: bool
 
 const
-  IdType = "TEXT"
-  DataType = "BLOB"
-  TimestampType = "INTEGER"
-
   dbExt* = ".sqlite3"
-  tableTitle* = "Store"
+  tableName* = "Store"
+
+  idColName = "id"
+  dataColName = "data"
+  timestampColName = "timestamp"
+
+  idColIndex = 0
+  dataColIndex = 1
+  timestampColIndex = 2
```

Comment:
OTOH, these 3 consts probably should be removed completely. They make sense only for statements with `SELECT *` which are not used currently and can be replaced with `select id,name,timestamp` explicily listing all fields we want.

Reply:
Change made.

One or two of them can still be useful for `SELECT id FROM ...` and `SELECT id, data FROM ...`, but the utility is admittedly low.

Reply:
I want to emphasize that indexes in `select *` and `select id` statements are different and even if they both equal the same 0 value - it's mere coincedence.

So if you have `select id`, you need to define index alongside this SQL text, rather than rely on hope to use the same field order as in `create` statement somewhere else and especially in a different module.